### PR TITLE
Adding E-Planning Adserver Support

### LINKED ISSUE #3026: Intent to implement: `amp-ad` type: 'epl-display'
Hi all,

I'd like to request the addition of the epl-display type for the amp-ad component.
I think that I've read all documentation there is to read on this topic, but please tell me if I'm missing something.

I already submit an Individual CLA and I am waiting for a response.

Thanks!

Sebastian

### PR DESCRIPTION
**Pull request by sebaperez**
Closes #3026.

### CODE REVIEW

File: ads/eplanning.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# e-planning
+
```
Comment (dvoytenko) [inline ads/eplanning.md]: Could you please add the reference to your service here? E.g. http://www.e-planning.net/?

**Comment by dvoytenko** [comment]
Thanks! Looks good. I just had one small request.

**Comment by dvoytenko** [comment]
Thanks! LGTM. Will merge as soon as it's green.

**Comment by sebaperez** [comment]
Thank you @dvoytenko !

**Pull request merged**